code/datums/diseases/xeno_transformation.dm

```diff
@@ -55,6 +55,8 @@
 					affected_mob.drop_inv_item_on_ground(items)
 				var/mob/living/carbon/xenomorph/drone/xeno = new(turf)
 				xeno.set_hive_and_update(XENO_HIVE_MUTATED)
+				if(length(xeno.hive.totalXenos) <= 1)
```

Comment:
Isnt this disabling queen evolve for the entire hive ??

Reply:
Yes, if the hive has no members other than the DDI xeno. Not really a big deal, won't affect events where admins make the base hive mutated. Could possibly affect thunderdome I guess?